### PR TITLE
tokio: make 1.20.x an LTS release

### PR DESCRIPTION
**Pull request by Darksonn**
This adds a new LTS release and removes an expired one.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -213,7 +213,7 @@ To use a fixed minor version, you can specify the version with a tilde. For
 example, to specify that you wish to use the newest `1.14.x` patch release, you
```
Comment (hds) [inline README.md:213]:
```suggestion
example, to specify that you wish to use the newest `1.18.x` patch release, you
```

File: tokio/README.md [hunk 2]
```diff
@@ -213,7 +213,7 @@ To use a fixed minor version, you can specify the version with a tilde. For
 example, to specify that you wish to use the newest `1.14.x` patch release, you
```
Comment (hds) [inline tokio/README.md:213]:
```suggestion
example, to specify that you wish to use the newest `1.18.x` patch release, you
```

**Review by hds — CHANGES_REQUESTED** [review]
There are a couple of extra places where `1.14` is mentioned.

**Review by hds — APPROVED** [review]
Looks good.

**Review by Noah-Kennedy — APPROVED** [review]

**Pull request merged**